BUILD.md

```diff
@@ -677,11 +677,6 @@ NOTE: To force the OSX version set the environment variable [MACOSX_DEPLOYMENT_T
 
 Setup Homebrew and components
 
-- Follow instructions on [brew.sh](http://brew.sh) to get Homebrew installed.
-
-      /usr/bin/ruby -e "$(curl -fsSL \
-          https://raw.githubusercontent.com/Homebrew/install/master/install)"
```

Comment:
These instructions are out of date. Trying to run this won't work now.

https://raw.githubusercontent.com/Homebrew/install/master/install